tests/Livewire/Order/OrderPositionsTest.php

```diff
@@ -146,4 +140,44 @@ public function test_can_show_related_columns()
         $this->assertArrayHasKey('order.uuid', $component->get('data.data.0'));
         $this->assertEquals($this->order->uuid, $component->get('data.data.0')['order.uuid']);
     }
+
+    public function test_quick_add_order_position()
+    {
+        $product = Product::factory()
+            ->for(VatRate::factory())
+            ->has(
+                Price::factory()
+                    ->for(
+                        PriceList::factory()->state([
+                            'is_default' => true,
+                            'is_net' => false,
+                        ])
+                    ),
+                'prices'
+            )
+            ->create();
+
+        $orderPositionCount = $this->order->orderPositions()->count();
+        Livewire::test(OrderPositions::class, ['order' => $this->orderPositionForm])
+            ->set('orderPosition.product_id', $product->id)
+            ->call('quickAdd')
+            ->assertStatus(200)
+            ->assertHasNoErrors()
+            ->assertReturned(true);
+
+        $this->assertEquals($orderPositionCount + 1, $this->order->orderPositions()->count());
+        $newOrderPosition = $this->order->orderPositions()->latest()->first();
+        $this->assertNotEquals(
+            $product->prices->first()->getNet($product->vatRate->rate_percentage),
```

Comment:
why not set a variable to the result of getNet() and getGross()? Then you don't need to call these methods again later.